packages/components/src/Modal/GenericModal/GenericModal.module.scss

```diff
@@ -0,0 +1,114 @@
+@import "~@kaizen/design-tokens/sass/shadow";
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/layout";
+@import "../../../styles/legacy/layout";
+@import "../../../styles/responsive";
+@import "../../../styles/layers";
+@import "../../../styles/border";
+@import "../../../styles/animation";
+
+.defaultModalWidth {
+  min-width: 300px;
+  max-width: 600px;
+}
+
+.backdropLayer {
+  @include ca-position($start: 0, $end: 0, $top: 0, $bottom: 0);
+
+  position: fixed;
+  background-color: #000;
+  opacity: 50%;
+  z-index: $ca-z-index-modal-backdrop;
+}
+
+.scrollLayer {
+  @include ca-position($start: 0, $end: 0, $top: 0, $bottom: 0);
+
+  position: fixed;
+  display: flex;
+  align-items: center;
+  z-index: $ca-z-index-modal;
+  overflow-y: auto;
+}
+
+.modalLayer {
+  // use flexbox auto margins to horizontally and vertically centre the modal
+  // because in IE11, `align-items: center` and `justify-content: center` cause
+  // long modal content to become unreachable if they go off-screen
+  margin: auto;
+  padding: $ca-grid 0;
+  width: 100%;
+}
+
+%genericModal {
+  margin: auto;
+  background-color: $color-white;
+  border-radius: $border-solid-border-radius;
+  box-shadow: $shadow-large-box-shadow;
+  position: relative;
+  width: 90%;
+  -webkit-font-smoothing: antialiased;
+  color: $color-purple-800;
+
+  @include ca-media-tablet-and-up {
+    width: 100%;
+  }
+}
+
+// WAITING FOR TEXTFIELD
+.animatingEnter {
+  transition-duration: $ca-duration-fast;
+
+  .backdropLayer {
+    /*
+      There's a weird bug in Chrome/Blink 102.x that animates the opacity
+      all the way to 1 with 200ms ("rapid" token), but is fine with
+      201ms, see Jira [KDS-523]
+    */
+    @include ca-animation-fade($duration: 201ms, $from: 0, $to: 0.5);
+  }
+
+  [data-modal] {
```

Comment:
Using this as a workaround, since converting `composes` to a placeholder broke the original functionality. [Source](https://github.com/cultureamp/kaizen-legacy/blob/a6e89ef68ad4af470c57af40c8c7b04a8d7bc4f9/draft-packages/modal/KaizenDraft/Modal/Primitives/GenericModal.module.scss#L71).